datapreview.tex

```diff
@@ -105,7 +109,7 @@ \subsection{Data Release 1}
 \centering
 \includegraphics[width=0.9\linewidth]{figures/DP1-products}
 \caption{Summary of data products expected in DP1.
-Note the high degree of uncertainty in this table. DP1 will be planned in detail during 2024.}
+Note the high degree of uncertainty in this table. DP1 will be planned in detail during 2024. \TODO{red}{NEEDS REVISION}}
```

Comment:
I see the "NEEDS REVISION" but since the graphics here explicitly say "LSSTCam" perhaps it would help if the table caption were already updated to mention ComCam.

E.g.:

"DP1, which may be modified to be based on ComCam data, will be planned in detail during 2024."

Reply:
This is being updated still - the graphics require update

Reply:
Ah, OK.  I had the impression you were trying to get a new version out before the graphics could be updated.  My suggestion would only be relevant in that case.